src/base/portal.tsx

```diff
@@ -49,6 +49,5 @@ export function PortalChild({
   if (portalEl) {
     return ReactDOM.createPortal(children, portalEl);
   }
-
-  return <>{children}</>;
+  return null;
```

Comment:
Largest change is here - what should happen if portalEl is undefiend in PortalChild? I think just not rendering anything makes sense, but happy to discuss.